FlaskApp/templates/control_panel.html

```diff
@@ -4,4 +4,5 @@
     <h1>Control Panel</h1>
     
     <a href="/admin/blog/new">New Blog Post</a>
+    <a href="/admin/blog/view_all">View all Blog Database Table rows</a>
```

Comment:
You might want to separate all of these links into their own <p> tags or <div> tags, so they're not all on the same line. Remember that <a> tag doesn't do a newline for you.